lib/LedgerEventStorage.js

```diff
@@ -142,6 +142,37 @@ class LedgerEventStorage {
     return {dupHashes};
   }
 
+  /**
+   * Filter a list of eventHashes.
```

Comment:
```Filter a list of `eventHashes` according to `blockHeight` or `consensus`.```

Reply:
Ok, I just remember reading the function description and thinking I had no idea how the event hashes were being filtered. I see that the params say "filter on". I guess we can leave it. Alternatively, we could say:

```Filter a list of `eventHashes` according to the given parameters.```

Do you think we need to mention, in the doc section, that you can't provide more than one parameter at a time?

Reply:
``` js
/**
 * Filter a list of eventHashes. The events can be filtered by either the `blockHeight` or the `consensus` parameters, but not both.
 *
 * @param {ObjectType} eventHash - a string or array of hashes to filter.
 * @param {ObjectFoo} [blockHeight] - filter on `meta.blockHeight`.
 * @param {ObjectBar} [consensus] - filter on `meta.consensus`.
 * ...
 */
``` 

Square brackets around the name indicate that the syntax is optional. The description can be used to describe the semantics of the parameters.

Reply:
@apowers313 thanks for guidance.